testing/scripts/playwright_test.py

```
@@ -17,6 +17,16 @@
 import requests as _requests
 from playwright.async_api import async_playwright, Page, Frame
 
+# Visual regression utilities (optional — only used in test_visual_regression)
+try:
+    import sys as _sys
+    import os as _os
+    _sys.path.insert(0, _os.path.dirname(__file__))
+    from snapshot_utils import capture_snapshot, compare_snapshot, set_viewport as _set_viewport
+    _SNAPSHOTS_AVAILABLE = True
+except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)